exercises/concept/paolas-prestigious-pizza/.meta/Exemplar.elm

```diff
@@ -0,0 +1,92 @@
+module PaolasPrestigiousPizza exposing
+    ( Pizza
+    , ingredientsParser
+    , menuParser
+    , oneIngredientParser
+    , pizzaParser
+    , priceParser
+    , vegetarianParser
+    , wordParser
+    )
+
+import Parser exposing ((|.), (|=), Parser)
+
+
+type alias Pizza =
+    { name : String
+    , vegetarian : Bool
+    , ingredients : List String
+    , price : Int
+    }
+
+
+priceParser : Parser Int
+priceParser =
+    Parser.int
+        |. Parser.symbol "€"
+
+
+vegetarianParser : Parser Bool
+vegetarianParser =
+    Parser.oneOf
```

Comment:
Is this defiinitely the best way of doing this? It looks a bit strange ...

Reply:
That's how in was in the original exercise.
In the hints, I suggested that `succeed` could be used too
```elm
Parser.suceed True |. Parser.symbol "(v)"
```
But I needed somewhere to suggest the use of `map`.

Now I'm thinking we could use map to capitalize the pizza names or something. What do you think?

Reply:
Or did you mean the use of `oneOf` for something that feels like an `optional` or `maybe` or something? They don't exist in the library.

Reply:
I think the clearest way to write it is the way you mention.

I might even change it to `Parser.symbol "(v)" |> andThen |> succeed True`, which makes the reading order the same as the logical order (not totally sure it works though!). And maybe isn't idiomatic elm parsing.

I think it is probably worth coming up with a different example for map, as this one isn't very compelling. Capitalizing the pizza names seems like a good shout for this ...

Reply:
Done.
I think I've addressed everything, unless I've missed something?

Reply:
Good for me!

Reply:
Although actually I can't see that the `Parser.suceed True |. Parser.symbol "(v)"` change has been made?

Reply:
It's been made, maybe you need to refresh?

Reply:
Ok, I can see it. The link above this comment goes to the specific commit rather than HEAD of the PR. You learn something new every day ...